src/deion/worker/core/player/value.js

```diff
@@ -58,37 +66,51 @@ const value = (
 	const slope = 1.531;
 	const intercept = 31.693;
 
-	// 1. Account for stats (and current ratings if not enough stats)
+	// 1. OVERALL RATINGS + PER + CONSISTENCY
+	// Account for stats (and current ratings if not enough stats)
 	const ps = p.stats.filter(playerStats => !playerStats.playoffs);
 	let current = pr.ovr; // No stats at all? Just look at ratings more, then.
 	if (process.env.SPORT === "basketball" && ps.length > 0) {
 		const ps1 = ps[ps.length - 1]; // Most recent stats
-
-		if (ps.length === 1 || ps[0].min >= 2000) {
+		if (ps.length === 1) {
 			// Only one year of stats
-			current = intercept + slope * ps1.per;
-			if (ps1.min < 2000) {
-				current = (current * ps1.min) / 2000 + pr.ovr * (1 - ps1.min / 2000);
+			if (ps1.min > 14 * ps1.gp) {
+				//high mpg
+				current = 0.6 * current + 0.4 * ps1.per;
```

Comment:
I like using more ratings and less stats. Not sure what exactly the balance should be, but I'd been meaning to try weighting ovr higher. I'm a little confused by your implementation though, why do it differently in "one year of stats" than "two most recent seasons"? Like why not use `intercept` and `slope` here too to estimate how PER maps to ovr?